### ISSUE TITLE
Tutorial/learning videos

### ISSUE THREAD

**Issue by Teknia** [SparkDevNetwork/Rock#1148]
Hi,

Could you please, please upload your videos to something like your youtube channel or preferably somewhere where I can download and watch the videos offline. There seems to be hardly any buffering on the current player so it is extremely painful to get through the shortest of videos if you are on a bit of a slow internet line.

Please. It is even hard to follow what the person is sometimes trying to say as things get chopped into hundreds of pieces.

Thanks.

**Comment by arranf** [comment]
This isn't really the place for this so I'm going to close this.
